src/index.ts

```diff
@@ -194,4 +198,31 @@ export default class BugzillaAPI {
 
     return commentStatus.id;
   }
+
+  public async createBug(bug: CreateBugContent): Promise<number> {
+    let bugStatus = await this.link.post("bug", object(CreatedBugSpec), bug);
+
+    if (!bugStatus) {
+      throw new Error("Failed to create bug.");
+    }
+
+    return bugStatus.id;
+  }
+
+  public async updateBug(
+    bugIdOrAlias: number | string,
+    data: UpdateBugContent,
+  ): Promise<any> {
```

Comment:
Can we put an actual type here, not just `any`?

Reply:
Ohh, I forgot to change it to the correct value, fixed.